### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by Gouiaa** [XinArkh/VNect#2]
Hello, @XinArkh  and Thanks for sharing your implementation. Actually, I am implementing this paper and I think the order of the concatenation is not correct w.r.t the paper (See fig5 in the paper)
https://github.com/XinArkh/VNect/blob/cd0a58f6bcbb354c608f345404213e007030e942/vnect_model.py#L146-L147.

I think should be replaced like this:

`self.res5c_branch2a_feat = tf.concat([self.res5c_delta_x, self.res5c_delta_y, self.res5c_delta_z, self.bn5c_branch2a, self.res5c_bone_length],`.

I want also to ask you if you can share the training script?

**Comment by XinArkh** [comment]
Hi! Thank you for your enthusiasm!

I have checked it again and found it a bit of misleading in figure 5.

In this implementation, the order of the component being contacted is the same as that in the original caffe code provided by the author:

```
layer {
  name: "res5c_branch2a_feat"
  type: "Concat"
  bottom: "res5c_branch2a"
  bottom: "res5c_delta_x"
  bottom: "res5c_delta_y"
  bottom: "res5c_delta_z"
  bottom: "res5c_bone_length"
  top: "res5c_branch2a_feat"
  concat_param {
    axis: 1
  }
}
```

(line 1212 - line 1224)

Maybe it would be helpful if you contact the author to ask for the original model for further research.

As for the training script: I'm so sorry but I don't have it either. Looking forward to your contribution!

**Comment by Gouiaa** [comment]
@XinArkh Thanks for your answer. I have contacted the authors many times and they did not answer me. I have some questions and requests: 
1) Did you gate the 30 frames per second ?
2) Could you share the code (I know maybe your not allowed to do that)?
3) Could you share the caffe weights?  
Thanks

**Comment by XinArkh** [comment]
That's a good question. I guess the author made a mistake calculating the scale number in figure 5, otherwise it is contradictory to the source code.
I can get 10~12 fps in my lab computer with a gtx1060.
I'm afraid I'm not permitted to give you the model directly. I remember I have experienced this too. The author replied to me several weeks after I requested. They just deal with plenty of request mails at one time. Be patient and good luck!
